docs/source/admin/traffic_vault.rst

```diff
@@ -79,6 +79,53 @@ Administration of the PostgreSQL database for Traffic Vault
 
 Similar to administering the Traffic Ops database, the :ref:`admin <database-management>` tool should be used for administering the PostgreSQL Traffic Vault backend.
 
+.. program:: reencrypt
+
+app/db/reencrypt/reencrypt
+--------------------------
+The :program:`reencrypt` binary is used to re-encrypt all data in the Postgres Traffic Vault with a new base64-encoded AES key.
+
+.. note:: For proper resolution of configuration files, it's recommended that this binary be run from the ``app/db/reencrypt`` directory.
+
+Usage
+"""""
+``./reencrypt [options]``
+
+Options and Arguments
+"""""""""""""""""""""
+.. option:: --new-key NEW_KEY
+
+	(Optional) The file path for the new base64-encoded AES key. Default is ``/opt/traffic_ops/app/conf/new.key``.
+
+.. option:: --previous-key PREVIOUS_KEY
+
+	(Optional) The file path for the previous base64-encoded AES key. Default is ``/opt/traffic_ops/app/conf/aes.key``.
+
+.. option:: --cfg CONFIG_FILE
+
+	(Optional) The path for the configuration file. Default is ``./reencrypt.conf``.
+
+.. option:: --help
+
+	(Optional) Print usage information and exit.
+
+.. code-block:: bash
+	:caption: Example Usage
+
+	 ./reencrypt --new-key ~/exampleNewKey.txt --previous-key ~/exampleOldKey.txt
+
+reencrypt.conf
+""""""""""""""
+This file deals with configuration of the Traffic Vault Database to be used with the :program:`reencrypt` tool.
+
+:dbname: The name of the PostgreSQL database used.
+:hostname: The hostname (:abbr:`FQDN (Fully Qualified Domain Name)`) of the server that runs the Traffic Vault Database.
+:password: The password to use when authenticating with the Traffic Vault database.
+:port: The port number (as a string) on which the Traffic Vault Database is listening for incoming connections (NOTE: the PostgreSQL default is 5432).
+:ssl: A boolean that sets whether or not the Traffic Vault Database encrypts its connections with SSL.
+:user: The name of the user as whom to connect to the database.
+
+
```

Comment:
This all looks correct. Unfortunately, due to things you didn't change, the docs are having some issues:

```
 docs/source/admin/traffic_vault.rst#L172

Title level inconsistent:
```

etc.

The problem is that the existing structure is using:

```rst
**************
Document Title
**************

Section Header
==============

Subsection Header
-----------------

Sub-subsection Header
^^^^^^^^^^^^^^^^^^^^^

Aside
"""""
```

which isn't following the guidelines I linked in my last review. You can either change your <kbd>"</kbd> heading underlines to <kbd>^</kbd>, or fix the document to conform to the guidelines. The document should be updated, but it's not related to the work your doing, so you don't have to.